django/db/models/sql/compiler.py

```diff
@@ -1115,7 +1117,7 @@ def as_sql(self):
             raise EmptyResultSet
         sql, params = [], []
         for annotation in self.query.annotation_select.values():
-            agg_sql, agg_params = self.compile(annotation)
+            agg_sql, agg_params = self.compile(annotation, select_format=True)
```

Comment:
Since you're changing this line already, maybe it's worth changing the variable names from agg_{sql,params} to something like a_sql or expression_sql.